tasks/main.yml

```diff
@@ -6,15 +6,15 @@
     - "../vars/empty.yml"
   tags: [always]
 
-- import_tasks: install.yml
+- import_tasks: install_apt.yml
   when: ansible_pkg_mgr == "apt"
   tags: [postgresql, postgresql-install]
 
 - import_tasks: install_yum.yml
   when: ansible_pkg_mgr == "yum" and ansible_distribution == "RedHat" 
```

Comment:
Please add change to when: ansible_pkg_mgr == "yum" and ansible_distribution == "RedHat" or ansible_distribution == "CentOS"
or just: when: ansible_pkg_mgr == "yum"

Reply:
@NissesSenap, @gclough or change to `ansible_os_family == 'RedHat'`